package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start": "webpack-dev-server",
-    "serve": "pm2-dev start server/index.js",
+    "serve": "./node_modules/.bin/eslint server/* && pm2-dev start server/index.js ",
```

Comment:
move lint to separate script for the time being - currently you have no ability to just lint your code, and linting is not available in watch mode, am I correct?